### PR TITLE
Benji/path utils update

### PR DESCRIPTION
**Pull request by BenjiPestell**
Fix path Utils

### CODE REVIEW

File: src/asmcnc/core_UI/test_path_utils.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import unittest
+from path_utils import get_path
+
+class TestPathUtils(unittest.TestCase):
```
Comment (archiepeterjarvis) [inline src/asmcnc/core_UI/test_path_utils.py:4]: beautiful !!

Comment (archiepeterjarvis) [reply src/asmcnc/core_UI/test_path_utils.py:4]: but please move to tests directory ;)

Comment (BenjiPestell) [reply src/asmcnc/core_UI/test_path_utils.py]: Done :)

File: tests/test_path_utils.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+sys.path.insert(0, '../src/asmcnc/core_UI')
+from path_utils import get_path
+
+class TestPathUtils(unittest.TestCase):
+
+    def test_get_path_files_only(self):
+        target_dir = "router_machine.py"
+        expected_result = "/easycut-smartbench/src/asmcnc/comms/router_machine.py"
+        result = get_path(target_dir, files_only=True).replace("\\", "/")
```
Comment (Masc999) [inline tests/test_path_utils.py:13]: the replace should not be needed. get_path and search_tree both take care.

Comment (BenjiPestell) [reply tests/test_path_utils.py]: Fixed :)

**Pull request merged**